CSA.md

```diff
@@ -1,215 +1,221 @@
+    10.1 <span class="header_3">Protection by Provider.</span>  <span class="keyterms_link">Provider</span> will indemnify, defend, and hold harmless <span class="keyterms_link">Customer</span> from and against all <span class="keyterms_link">Provider Covered Claims</span> made by someone other than <span class="keyterms_link">Customer</span>, <span class="keyterms_link">Customer's</span> Affiliates, or Users, and all out-of-pocket damages, awards, settlements, costs, and expenses, including reasonable attorneys' fees and other legal expenses, that arise from the <span class="keyterms_link">Provider Covered Claim</span>.
 
-    10.2 <span class="header_3">Protection by Customer</span>.  <span class="coverpage_link">Customer</span> will defend and hold harmless <span class="coverpage_link">Provider</span> from and against all <span class="coverpage_link">Customer</span> Covered Claims made by someone other than <span class="coverpage_link">Provider</span> or its Affiliates, and indemnify <span class="coverpage_link">Provider</span> for all out-of-pocket damages, awards, settlements, costs, and expenses, including reasonable attorneys' fees and other legal expenses, that directly result from the <span class="coverpage_link">Customer</span> Covered Claim.
+    10.2 <span class="header_3">Protection by Customer</span>.  <span class="keyterms_link">Customer</span> will indemnify, defend, and hold harmless <span class="keyterms_link">Provider</span> from and against all <span class="keyterms_link">Customer Covered Claims</span> made by someone other than <span class="keyterms_link">Provider</span> or its Affiliates, and all out-of-pocket damages, awards, settlements, costs, and expenses, including reasonable attorneys' fees and other legal expenses, that arise from the <span class="keyterms_link">Customer Covered Claim</span>.
 
     10.3 <span class="header_3">Procedure.</span>  The Indemnifying Party's obligations in this section are contingent upon the Protected Party: (a) promptly notifying the Indemnifying Party of each Covered Claim for which it seeks protection; (b) providing reasonable assistance to the Indemnifying Party at the Indemnifying Party's expense; and (c) giving the Indemnifying Party sole control over the defense and settlement of each Covered Claim. A Protected Party may participate in a Covered Claim for which it seeks protection with its own attorneys only at its own expense. An Indemnifying Party may not agree to any settlement of a Covered Claim that contains an admission of fault or otherwise materially and adversely impacts the Protected Party without the prior written consent of the Protected Party.
```

Comment:
Grammar fix, "An" to "The" in the last sentence

```suggestion
    10.3 <span class="header_3">Procedure.</span>  The Indemnifying Party's obligations in this section are contingent upon the Protected Party: (a) promptly notifying the Indemnifying Party of each Covered Claim for which it seeks protection; (b) providing reasonable assistance to the Indemnifying Party at the Indemnifying Party's expense; and (c) giving the Indemnifying Party sole control over the defense and settlement of each Covered Claim. A Protected Party may participate in a Covered Claim for which it seeks protection with its own attorneys only at its own expense. The Indemnifying Party may not agree to any settlement of a Covered Claim that contains an admission of fault or otherwise materially and adversely impacts the Protected Party without the prior written consent of the Protected Party.
```